src/api/sockets/ZProxySocketController.js

```diff
@@ -78,14 +92,25 @@ class ZProxySocketController {
 
     pushToClients(msg) {
         if (this.wss) {
-            this.wss.connections.forEach(client => {
-                if (client.state === 'open') {
+            this.wss.clients.forEach(client => {
+                if (client.readyState === 1) {
```

Comment:
Let's use named constant instead. It is more verbose.

Reply:
I assume you mean using the `WebSocket.OPEN` enum instead of the int `1`. But should we use an enum in a JS file?